op-node/rollup/types.go

```diff
@@ -393,6 +396,11 @@ func (c *Config) GetPayloadVersion(timestamp uint64) eth.EngineAPIMethod {
 	}
 }
 
+// IsPlasmaEnabled returns false for now until we add the DA challenge contract parameters.
```

Comment:
This should be the final function doc comment. If you want to leave a TODO, use TODO(issue number) and create a new issue.